change/@ni-nimble-angular-d3bfc04f-a6c1-43ac-9d63-8424dd3c1343.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
from a text search in the branch, some comment references:

https://github.com/ni/nimble/blob/cc3e35b0f8883ca05625f41e3b8cf05f33e7db91/angular-workspace/projects/ni/nimble-angular/mapping/text/nimble-mapping-text.directive.ts#L10

https://github.com/ni/nimble/blob/cc3e35b0f8883ca05625f41e3b8cf05f33e7db91/packages/nimble-components/src/table-column/enum-base/models/table-column-enum-base-validator.ts#L13